process/graduation_criteria.md

```diff
@@ -0,0 +1,118 @@
+# Azure Container Upstream Graduation Criteria v0.1
+The Azure Container Upstream Graduation Criteria owes much of its basis on two existing documents,
+[CNCF Graduation Criteria](https://github.com/cncf/toc/blob/fc161c13a86ba022277a906a10ff51a568406f7c/process/graduation_criteria.adoc) 
+and [.NET Foundation Project Maturity Profile](https://github.com/dotnet-foundation/project-maturity-model/blob/65fdfa43d7e68845bd65638c378cf834d7158b90/maturity-profiles.md). The goal of this document is to describe the process and 
+minimum bar for each increasing level of project maturity to both provide a
+goal for the project maintainers, but also to set expectations for consumers of these
+projects.
+
+## Process
+
+Each project in Container Upstream (AzCU) has an associated maturity level. A proposed AzCU project 
+should enter at the project's preferred maturity level. If the project originates within the AzCU 
+group, then it should begin at the Sandbox Stage. If a joining project originates outside of the 
+AzCU group, the preferred maturity level of the project must meet a 2/3 supermajority vote by the 
+AzCU technical oversight committee (TOC). If the vote tally can not be met at the preferred 
+maturity level, each previous level will receive a vote until a 2/3 supermajority is met. If the 
+project does not receive enough votes at the Sandbox stage, the project is rejected.
```

Comment:
As we build out the governance. It might be a good point to change "rejected" to something around telling people what is missing from their proposal/project etc. That way people will know i can not re-ask for votes until i meet xyz. 

I am trying to preempt governing by "technicality of how the proposal looks", "bike-shedding"  etc..

Reply:
> the project is rejected with an opinion authored by the TOC